### PR TITLE
x11/gb18030: Update for gb18030-2022 certification

### PR DESCRIPTION
**Pull request by zhaocongjia**
Note: the SLE test needle is marked as soft-failure because of https://bugzilla.suse.com/show_bug.cgi?id=1216805.

- Related ticket: https://progress.opensuse.org/issues/150968
- Needles: Updated
- Verification run:
SLE: https://openqa.suse.de/tests/12832665#step/gb18030/89
TW: https://openqa.opensuse.org/tests/3734204#step/gb18030/91

### CODE REVIEW

File: tests/x11/gb18030.pm [hunk 1]
```diff
@@ -65,7 +65,7 @@ sub run {
     # the following fonts preparing steps are documented at:
     # https://confluence.suse.com/display/~kailiu/How+to+prepare+the+system+for+GB18030-2005+certification
     become_root;
-    zypper_call("rm arphic-ukai-fonts arphic-uming-fonts baekmuk-*-fonts noto-*-tc* noto-*-jp* noto-*-kr* wqy* adobe-sourcecodepro-fonts xorg-x11-fonts-converted", exitcode => [0, 104]);
+    zypper_call("rm arphic-ukai-fonts arphic-uming-fonts baekmuk-*-fonts noto-*-tc* noto-*-jp* noto-*-kr* wqy* adobe-sourcecodepro-fonts xorg-x11-fonts-converted xscreensaver", exitcode => [0, 104]);
```
Comment (GraceWang571) [inline tests/x11/gb18030.pm:68]: Why do we need to remove xscreensaver? To keep test case more robust?

**Comment by GraceWang571** [comment]
I see the gb18030-test-data-original.zip for 2005 version is under the data folder. Can you please also add the latest "18030-2022 Testing Sample-1026.zip" to keep consistent? Thanks.

File: tests/x11/gb18030.pm (hunk 1 above)
Comment (zhaocongjia) [reply tests/x11/gb18030.pm:68]: Yes, xscreensaver installs fonts that affects characters in "user defined region 1".

**Comment by zhaocongjia** [comment]
zip file update, good catch!

File: tests/x11/gb18030.pm (hunk 1 above)
Comment (GraceWang571) [reply tests/x11/gb18030.pm:68]: Sorry, I am not quite clear about this. 
Does this mean if a user wants to use the characters in "user defined region 1" smoothly, the xscreensaver must be removed from the system?

Comment (GraceWang571) [reply tests/x11/gb18030.pm:68]: Thanks for your explanation. 
LGTM now.

**Pull request merged**